lib/editable.js

```diff
@@ -0,0 +1,124 @@
+var $ = require('jquery');
+
+var Exercises = require('./exercises');
+var Food = require('./foods');
+var Breakfast = require('./breakfast');
+var Lunch = require('./lunch');
+var Dinner = require('./dinner');
+var Snacks = require('./snacks');
+var Exercise = require('./exercise')
+
+function Editable () {};
+
+Editable.prototype.make = function(table) {
```

Comment:
@MDes41 Nice refactor, I'm going to need to look at this more closely to pick up the technique!